packages/prisma/migrations/20231027144522_add_ical_columns_to_booking/migration.sql

```diff
@@ -0,0 +1,9 @@
+/*
+  Warnings:
+
+  - Added the required column `iCalUID` to the `Booking` table without a default value. This is not possible if the table is not empty.
+
+*/
+-- AlterTable
+ALTER TABLE "Booking" ADD COLUMN     "iCalSequence" INTEGER NOT NULL DEFAULT 0,
```

Comment:
These adds are out of order compared to the code. Intentional?

Reply:
To clarify - The order specified throughout the code is `iCalUID` and then `iCalSequence`.  See prisma schema.

This alter table statement has them in the reverse order.

Reply:
This file was generated using `prisma migrate`. I can fix this and moving forward ensure that the adds line up with the schema.